### PR TITLE
Redraft for tone and simple language

### PR DESCRIPTION
**Pull request by rhiaro**
***
<a href="https://pr-preview.s3.amazonaws.com/w3ctag/the-web-is-not-versioned/pull/3.html" title="Last updated on Jan 23, 2024, 4:46 PM UTC (fd76a16)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3ctag/the-web-is-not-versioned/3/fee3574...fd76a16.html" title="Last updated on Jan 23, 2024, 4:46 PM UTC (fd76a16)">Diff</a>

### CODE REVIEW

**Review by torgo — APPROVED** [review]
lgtm

File: index.html [hunk 1]
```diff
@@ -49,82 +49,58 @@
...
 ## Conclusion
 
-For the above-mentioned reasons,
-we believe that any trend towards versioning the web is harmful,
-and any versioning happening out in the wild is unsanctioned and harmful.
-There is [one web](https://www.w3.org/TR/ethical-web-principles/#oneweb).
+It is important to develop the web platform in line with the principle that
+there is [one web](https://www.w3.org/TR/ethical-web-principles/#oneweb).
+Versioning should be constrained to individual specifications and features,
+and not applied to the web platform as a whole.
```
Comment (cynthia) [inline index.html]:
```suggestion
```

Versioning of specifications is a model we would also slowly want to move away from.

File: index.html [hunk 2]
```diff
@@ -32,99 +32,67 @@
...
-As the web is a standard-governed platform,
-we only acknowledge proposals that fall under the first criteria.
-Associating a group of non-standardized technologies
-and claiming it as a specific version or evolution of the web
-effectively is going in the direction of the latter -
-and is false marketing, especially if your fork claims to be another web.
+while breaking as little as possible
+(ideally nothing - for example,
+through [polyfill prototyping](https://www.w3.org/2001/tag/doc/polyfills/)).
+The most effective way to do this is through discrete, well-isolated contributions.
```
Comment (cynthia) [inline index.html]: We probably should have a simpler word to replace discrete.

File: index.html [hunk 3]
```diff
@@ -32,99 +32,67 @@
...
-that has interoperability guarantees.
+This allows the community to focus on the merits
+of each technical contribution to the web
+in a distributed and modular way. 
 
-It’s, therefore, inappropriate and potentially confusing
-to the ecosystem, for the web standards community
-to associate web feature sets
-with a version number in the traditional sense of software versioning.
+Verison numbers such as "Web 2.0" have been applied
```
Comment (cynthia) [inline index.html]:
```suggestion
Version numbers such as "Web 2.0" have been used
```

File: index.html [hunk 4]
```diff
@@ -32,99 +32,67 @@
...
+across the industry of web technologies for eductional,
```
Comment (cynthia) [inline index.html]:
```suggestion
across the industry of web technologies for educational,
```

File: index.html [hunk 5]
```diff
@@ -32,99 +32,67 @@
...
-of a collection of interlinked static documents.
-To counter this stagnation, there was a movement
-dubbed “Web 2.0” to rejuvenate the web
-into its next stage of evolution.
-This idea included concepts
-such as interactive web applications,
-arguably ushering in what we now think of as the modern web.
-However, the 2.0 moniker was not intended to be
-considered a version in the sense that software is versioned.
+Versioning is helpful for individual specifications or features,
```
Comment (cynthia) [inline index.html]:
```suggestion
Versioning may be helpful for individual specifications or features,
```

File: index.html [hunk 6]
```diff
@@ -32,99 +32,67 @@
...
-and suggesting it as a versioned evolution.
-We declare this as harmful
-and explain why this goes against the principles of the web.
+The web does not use versioning at a platform level,
+because it benefits from [continuous evolution](https://www.w3.org/2001/tag/doc/evergreen-web/).
 
-## The modern web does not need versioning
+The web platform is made up of many different technologies.
+A large community drives the development of these technologies,
+and some are dependant on each other, often in a complex way.
```
Comment (cynthia) [inline index.html]:
```suggestion
and some are dependent on each other.
```

**Review by cynthia — COMMENTED** [review]
I think we want to specifically call out bad patterns we have observed, but that could be just me.

**Pull request merged**